### PR TITLE
[kk] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@altynbek07, can you help us and review this automatic pull request, please ? thanks by advance

**Comment by caouecs** [comment]
I merge this machine translation, but it's possible to do a pull request after to fix words, see [contributing section](https://github.com/Laravel-Lang/lang/blob/master/docs/contributing-to-dev.md)

File: locales/kk/validation.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => ' :attribute тек әріптерден, сандардан және сызықшалардан тұруы керек.',
     'alpha_num'            => ' :attribute тек әріптерден және сандардан тұруы керек.',
     'array'                => ' :attribute жиым болуы керек.',
-    'attached'             => 'This :attribute is already attached.',
+    'attached'             => 'Бұл нөмір :attribute тіркелген.',
```
Comment (Lootjs) [inline locales/kk/validation.php:23]: 'Бұл нөмір :attribute тіркелген.',

Comment (caouecs) [reply locales/kk/validation.php:23]: Hello,
sorry, but I don't understand what is wrong

**Pull request merged**